<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>seatavailability</class>
 <widget class="QDialog" name="seatavailability">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CheckSeats</string>
  </property>
  <widget class="QComboBox" name="cmbselect">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select Train</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>229</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
color:black;
	
	border-radius:12px;
	background-color: rgb(240, 102, 84);
	
	border: 3px solid white;

}
QPushButton:hover {
    
	
	background-color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="txtTotalSeats">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>320</y>
     <width>71</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(251, 18, 19);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="txtBookedSeats">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>317</y>
     <width>80</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(251, 18, 19);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="txtAvailableseats">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>320</y>
     <width>71</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(251, 18, 19);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelofpic">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>720</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/PICS/seatspage.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);

background-color: rgb(1, 66, 96);
border-radius:15
</string>
   </property>
   <property name="text">
    <string>       SEATS</string>
   </property>
  </widget>
  <zorder>labelofpic</zorder>
  <zorder>cmbselect</zorder>
  <zorder>label</zorder>
  <zorder>pushButton</zorder>
  <zorder>txtTotalSeats</zorder>
  <zorder>txtAvailableseats</zorder>
  <zorder>txtBookedSeats</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
